content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md

```diff
@@ -86,6 +86,19 @@ Cipher is (NONE)
 
 ## Cleanup
 
+Delete sample applications `sleep` and `httpbin` from `foo` namespace:
+
+{{< text bash >}}
+$ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+$ kubectl delete -f samples/sleep/sleep.yaml -n foo
+{{< /text >}}
+
+Uninstall Istio from cluster:
```

Comment:
```suggestion
Uninstall Istio from the cluster:
```

Reply:
addressed as part of [363e5ae](https://github.com/istio/istio.io/pull/12735/commits/363e5aed9e6a27a8875701ad61e2e51bf6c03142)